src/main/groovy/com/palantir/gradle/javadist/JavaDistributionPlugin.groovy

```diff
@@ -58,19 +59,37 @@ class JavaDistributionPlugin implements Plugin<Project> {
             }
         }
 
+        Task copyLauncherBinaries = project.tasks.create('copyLauncherBinaries', {
+            group = GROUP_NAME
+            description = "Creates go-java-launcher binaries."
+        }) << {
+            // TODO(rfink) Don't check these in, but fetch them from a published artifact?
+            ["javalauncher-linux-amd64", "javalauncher-darwin-amd64"].each { file ->
+                def dest = Paths.get("${project.buildDir}/scripts/${file}")
```

Comment:
one more thought here, we can also split the configs between things that should not be editable, saved in service/bin and things that should be editable (such as java options) in var/launcher.